### PR TITLE
Trigger updates

### PR DESCRIPTION
**Pull request by lassem**
This seems to trigger some update events in the very least.
The OpenHAB UI is still not responding.

I will attach some logs to #11

### CODE REVIEW

File: addons/binding/org.openhab.binding.freeathome/src/main/java/org/openhab/binding/freeathome/handler/FreeAtHomeBridgeHandler.java [hunk 1]
```diff
@@ -337,7 +337,11 @@ private void connectGateway() {
         try {
             // Extract jID
             String jid = this.getJid(m_Configuration.login);
-            m_XmppClient.login(jid, m_Configuration.password);
+            m_XmppClient.login(jid.substring(0, jid.indexOf("@")), m_Configuration.password);
```
Comment (ruebox) [inline addons/binding/org.openhab.binding.freeathome/src/main/java/org/openhab/binding/freeathome/handler/FreeAtHomeBridgeHandler.java:340]: Is this a necessary to only used the login without busch-jaeger.de suffix?

File: addons/binding/org.openhab.binding.freeathome/src/main/java/org/openhab/binding/freeathome/handler/FreeAtHomeBridgeHandler.java [hunk 2]
```diff
@@ -337,7 +337,11 @@ private void connectGateway() {
...
+
+            Presence presence = new Presence(Jid.of("mrha@busch-jaeger.de/rpc"), Presence.Type.SUBSCRIBE, null, null, null, null, Jid.of(jid), null, null, null);
+            m_XmppClient.send(presence);
```
Comment (ruebox) [inline addons/binding/org.openhab.binding.freeathome/src/main/java/org/openhab/binding/freeathome/handler/FreeAtHomeBridgeHandler.java:343]: Ok. Here I subscribe for a presence @ mrha. Do you know, if presence update is continuously sent?

File: addons/binding/org.openhab.binding.freeathome/src/main/java/org/openhab/binding/freeathome/handler/FreeAtHomeBridgeHandler.java [hunk 3]
```diff
@@ -393,14 +397,14 @@ private void connectGateway() {
         // m_XmppClient.getManager(PresenceManager.class).requestSubscription(j, "");
         // m_XmppClient.getManager(PresenceManager.class).approveSubscription(j);
 
-        // Presence presence = new Presence(j);
+         Presence presence = new Presence();
```
Comment (ruebox) [inline addons/binding/org.openhab.binding.freeathome/src/main/java/org/openhab/binding/freeathome/handler/FreeAtHomeBridgeHandler.java:400]: Here an default presence is generated and sent at line 407. 
What happens if I sent a default presence or subscribe @ a specific user such as mrha?

File: addons/binding/org.openhab.binding.freeathome/src/main/java/org/openhab/binding/freeathome/handler/FreeAtHomeBridgeHandler.java (hunk 1 above)
Comment (ruebox) [reply addons/binding/org.openhab.binding.freeathome/src/main/java/org/openhab/binding/freeathome/handler/FreeAtHomeBridgeHandler.java:340]: ah ok, understood jid is required to subscribe for presence.

**Comment by lassem** [comment]
I added these two events to better match what the official webapp did.

**Pull request merged**